book/MovingToTheGPU.html

```diff
@@ -3896,5 +3896,4 @@
 <link rel='stylesheet' href='../style/book.css'>
 <style class="fallback">body{visibility:hidden;white-space:pre;font-family:monospace}</style>
 <script src="markdeep.min.js"></script>
-<script src="https://morgan3d.github.io/markdeep/latest/markdeep.min.js"></script>
```

Comment:
Since we can be sure that we'll include it in the deployment of the site we don't need to have it. It just is more efficient in caching the script for the entire site as well.

Reply:
Although I cannot be sure of this, you'll get a CORS error if you try to include the script tag and check your browser console. Hence, you are either required to add `crossorigin="anonymous"` to the script tag or have `markdeep.min.js` be included in the project. This might be an oversight by the author and having the script included with the project is probably more than enough and allows us to change it in the future.

Reply:
@hollasch Do you know if this has been an issue with RTW? It may by worth clarifying with Morgan since you're already in touch with him.

Personally, I agree with @KorigamiK in that this feels redundant since we already host our own copies of `markdeep.min.js`.

Reply:
Yes, it's _explicitly_ redundant. The idea is to pick up any Markdeep fixes/improvements immediately and automatically. We do update the backup copy in the project, but often with a lot of lag (since I'm not always monitoring Markdeep releases).

Reply:
> Do you know if this has been an issue with RTW?

I wouldn't say it's an problem, but I can't think of a reason not to adopt the recommended protocol, so I've added these throughout in the RTW project.

Reply:
OK, the versioning argument makes sense. In that case I'm inclined to leave this in to stay consistent with the rest of the RTW project.